dxfeed/core/pxd_include/EventData.pxd

```diff
@@ -163,7 +163,8 @@ cdef extern from "EventData.h":
         dxf_int_t raw_flags,
         dxf_char_t exchange_code,
         dxf_price_type_t day_close_price_type,
-        dxf_price_type_t prev_day_close_price_type
+        dxf_price_type_t prev_day_close_price_type,
+        dxf_order_scope_t scope
 
 
```

Comment:
In this file we are using imports using `*` - it's really confusing practice bcs after some time you have absolutely no idea from what particular file which function/Class came from. As mentioned in python Zen 😉 
>Namespaces are one honking great idea -- let's do more of those!